XLForm/XL/Cell/XLFormSelectorCell.m

```diff
@@ -266,6 +266,15 @@ -(void)formDescriptorCellDidSelectedWithFormController:(XLFormViewController *)c
             alertController.popoverPresentationController.sourceRect = [formViewController.tableView convertRect:v.frame fromView:self];
             __weak __typeof(self)weakSelf = self;
             for (id option in self.rowDescriptor.selectorOptions) {
+				NSString *optionTitle = [option displayText];
+				if (self.rowDescriptor.valueTransformer){
+					NSAssert([self.rowDescriptor.valueTransformer isSubclassOfClass:[NSValueTransformer class]], @"valueTransformer is not a subclass of NSValueTransformer");
+					NSValueTransformer * valueTransformer = [self.rowDescriptor.valueTransformer new];
+					NSString * transformedValue = [valueTransformer transformedValue:[option valueData]];
+					if (transformedValue) {
+						optionTitle = transformedValue;
```

Comment:
Sorry for forgetting to include the subsequent line that replaces using the displayText with optionTitle. I have included a second commit (f1f8a21bf075eb09cbde91d10527b97f2d425449) that fixes this. Thanks for reviewing @mats-claassen.